### ISSUE TITLE
Chrome Canary not available for Ubuntu 16.04

### ISSUE THREAD

**Issue by namvuCosmo** [webrtc/KITE#13]
The Canary version of Chrome is not currently available for ubuntu, will there be one in the future?

**Comment by huibk** [comment]
Not that I'm aware of. Chrome Dev is updated 1-2 week, and should be sufficiently bleeding edge for KITE testing.

**Comment by agouaillard-cosmo** [comment]
1. the document states canary on ubuntu so we wanted tot be sure you would be ok with something else.
2. we would then be testing dev on ubuntu and canary on every other OS. Can you confirm you are OK with that, please.

**Comment by huibk** [comment]
Yes, testing on dev instead of canary is fine. Dev is typically only a few days behind canary.
